misc_docs/syntax/operator_ref_assignment.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: "ref-assignment"
+keywords: ["ref", "assign"]
+name: ":="
+summary: "This is the `ref assignment` operator."
+category: "operators"
+---
+
+This operator is syntactic sugar for assigning to the `contents` value of a `ref`
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res
+let total = ref(0)
+total := 1
+```
+
+```js
+var total = {
+  contents: 0
+};
+
+total.contents = 1;
+```
+
```

Comment:
~~~suggestion
A `ref` is a builtin record type with a `mutable contents` field. The `:=` operator is just a shorthand version for the following code:

```res
let total = ref(0)
total.contents = 0
```
~~~